pages/windows/pathping.md

```diff
@@ -0,0 +1,35 @@
+# pathping
+
+> A trace route tool combining features of Ping and Tracert.
+
+- Ping and trace the route to a host:
+
+`pathping {{hostname}}`
+
+- Do not resolve addresses to host names:
```

Comment:
hostnames to addresses ?

Reply:
It's a bit odd, but it is addresses to hostnames.
![Screenshot](https://image.prntscr.com/image/bwzuuGQSTNOdvmmKSp4wyA.png)

Reply:
>Or maybe `-n` prevents the reverse lookup of the hostname for the corresponding IP?

That is correct. It prevents looking up the hostname from the IP. I'm probably just getting confused.

Reply:
Ok, then perhaps - `Do not perform reverse lookup of ip-address to hostname:` is more clear.

Reply:
Sounds good to me. :+1: